### PR TITLE
Create pull_request_template.md

### PR DESCRIPTION
**Pull request by MaksymilianDemitraszek**
### Checklist
- [x] Relevant issue is linked
- [x] Docs updated/issue for docs created
- [x] Added relevant tests

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+### Checklist
+- [ ] Relevant issue is linked
+- [ ] Docs updated/issue for docs created
+- [ ] Added relevant tests
```
Comment (cptartur) [inline .github/pull_request_template.md:4]: Alongside the checklist I'd include a "template" for linking issue so it's even harder to forget and so we follow the same format in every PR. Like

```suggestion
Closes 

### Checklist
- [ ] Relevant issue is linked
- [ ] Docs updated/issue for docs created
- [ ] Added relevant tests
```

**Review by Arcticae — APPROVED** [review]

File: .github/pull_request_template.md (hunk 1 above)
Comment (Arcticae) [reply .github/pull_request_template.md:4]: I don't think that's necessary, one might link with the UI directly

Comment (cptartur) [reply .github/pull_request_template.md:4]: Didn't think of the UI. If we prefer it for linking the issues, current template is fine.

**Review by cptartur — APPROVED** [review]

File: .github/pull_request_template.md (hunk 1 above)
Comment (MaksymilianDemitraszek) [reply .github/pull_request_template.md:4]: I prefer UI, there is a reminder about linking the issue already

**Pull request merged**